### DISCUSSION TITLE
Ability to limit number of homes in a certain dimension

### DISCUSSION THREAD

**Discussion by YanisBft** [EssentialsX/Essentials#4003] [Ideas and Feature Suggestions] [answered]
It would be nice to be able to limit the number of homes per dimension.
For instance, to let players set only 2 homes in the Nether.

**Comment by mdcfe** [comment, marked as answer]
This is unlikely to be added in 2.x due to the complications around not breaking existing setups. However, we're considering per-world limits for a future plugin version.
